frontend/test/pytest/test_debug.py

```diff
@@ -118,9 +119,14 @@ def test(_x):
     def test_intermediate_values(self, capfd, arg, expected):
         """Test printing of arbitrary JAX tracer values."""
 
+        if qml.capture.enabled():
+            loop_fn = qml.for_loop  # TODO: unify qml.for_loop and catalyst.for_loop
```

Comment:
I thought qml.for_loop was already dispatching to catalyst.for_loop with capture disabled.